articles/resources/relnotes.md

```diff
@@ -17,6 +17,17 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version X.XX.XXXXXXXX
+
+*Release date: November 23rd, 2020*
+
+- Improved compiler performance with faster reference loading.
+- Updated Q# grammar for language repository.Fixed lexer bug with double literals and ranges. Updated grammar for namespaces, namespace elements, statements, and types and labels are used consistently. 
+- Updated Microsoft.Quantum.Preparation API.
+- Q# ergonomic improvements: eliminate the need for parentheses around subsequent calls. 
+- Updated IQ# client telemetry to use the latest Microsoft Cookie Compliance API and GDPR cookie consent banner. This requires explicit user consent for non-critical cookies, and the user has a finer control on how cookies can be used. More here: https://www.1eswiki.com/wiki/WCP_Cookie_Consent_API.
```

Comment:
@vxfield, does the user actually have finer control over cookies in IQ# than they did before, since we are classifying the cookie as 'Required'?

Reply:
The cookie API itself provides that finer control, but in our case it does not change anything because we only use the cookies for audience counting (which is required) and don't use it for Ads, or anything else.